Lib/test/test_socket.py

```diff
@@ -75,12 +94,20 @@ def _have_socket_alg():
         s.close()
     return True
 
+def _have_socket_vsock():
+    """Check whether AF_VSOCK sockets are supported on this host."""
+    if (getCid() == None):
```

Comment:
@caavery read @berkerpeksag's suggestion as following:

```python
    foo = getCid is not None
    return foo
```
`foo` will contain a boolean value which will be returned.